### PR TITLE
Tiny edits made

### PR DESCRIPTION
**Pull request by karinakozarova**
Tiny typos updated after the previous PR.

### CODE REVIEW

File: resources.md [hunk 1]
```diff
@@ -11,9 +11,48 @@
...
+
+
 
 # Books
 <a href = "http://www.htmlandcssbook.com"> HTML and CSS book</a> <br>
 <a href = ""> </a> <br>
 <a href = ""> </a> <br>
 
+# Articles
+<a href ="https://medium.com/wd-tips-tricks/essential-html-css-and-javascript-tools-and-libraries-bbd39b4ed7a"> Essential libraries</a> <br>
```
Comment (nelsonic) [inline resources.md]: We prefer to use Markdown Syntax for links: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links 😉

File: resources.md [hunk 2]
```diff
@@ -11,9 +11,48 @@
...
+<a href = "https://medium.com/@joelennon/common-html-mistakes-de28db16b964"> Common HTML mistakes </a> <br>
+<a href = "https://medium.com/@narrowd/html-css-js-in-human-terms-36f0b4c35f6d"> JS, HTML, CSS in human terms </a> <br>
+<a href = "https://medium.com/@AlexDevero/what-every-web-designer-needs-to-know-about-html-tables-d02278c364da"> What every web designer needs to know about HTML tables</a> <br>
+<a href = "https://medium.com/dev-rocket/components-the-future-of-html-development-f692b5b6e22b"> The future of web development</a> <br>
+<a href = "https://medium.com/sam-dutton/high-performance-html-d40262388f29"> High performance HTML</a> <br>
+<a href = "https://medium.com/@janusboye/we-wanted-cat-videos-so-we-got-html5-4be1f93257d2"> We wanted cat videos so we got HTML5 </a> <br>
+# Project ideas
+
+<a href = "https://medium.freecodecamp.com/creating-a-bare-bones-quote-generator-with-javascript-and-html-for-absolute-beginners-5264e1725f08"> Quotes generator (HTML + JS)</a> <br>
+<a href = ""> </a> <br>
```
Comment (nelsonic) [inline resources.md]: empty links?

File: syntax(done and to do).md [hunk 3]
```diff
@@ -13,22 +13,9 @@
 
 ## TO DO:
 <ul> 
-
-<li> Create full syntax for HTML5
+<li> Create full syntax for HTML5 (created issues on the basic stuff that has to be added)
```
Comment (nelsonic) [inline syntax(done and to do).md]: link to the issue... 👍

**Review by nelsonic — APPROVED** [review]
@karinakozarova looks good. 
I've left a couple of comments, but these should not "_block_" merging. thanks again for your great work! 🎉

**Review by iteles — APPROVED** [review]
These are great, thanks again @karinakozarova ! 💃

**Pull request merged**